locales/de/forms.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
These should really be PkmnID => Form# => FormName.

This way when additional forms are added they can be added per pokemon (or in this case pokemon id).